fix(import/zsh-histdb): missing or wrong fields

Aloxaf opened the pull request:
<!-- Thank you for making a PR! Bug fixes are always welcome, but if you're adding a new feature or changing an existing one, we'd really appreciate if you open an issue, post on the forum, or drop in on Discord -->

This PR do two things:
1. Imports `exit_status` and `session` columns from zsh-histdb
2. Converts `duration` to nanoseconds for importing

## Checks
- [x] I am happy for maintainers to push small adjustments to this PR, to speed up the review cycle
- [x] I have checked that there are no existing pull requests for the same thing

ellie commented on atuin-client/src/import/zsh_histdb.rs:
```diff
@@ -76,12 +82,9 @@ impl From<HistDbEntry> for History {
-                    .to_string(),
-            );
+            .exit(histdb_item.exit_status)
+            .session(format!("{}:{}", hostname, histdb_item.session))
```
This isn't enforced (yet) by types, but by convention; Atuin uses UUIDs for the session ID. Specifically a uuidv7, but really any would work

Is it possible to ensure we do the same here?

The importer can group what it's importing by session, and then map that to a new session ID

Aloxaf commented:
> Just a couple of comments

Thanks for your suggestions. I have updated my code.

> Are there any other fields we are missing?

No more fields.
But I notice that `duration` should be measured in nanoseconds. Considering this is a minor bug, I also fixed it in this PR.

ellie reviewed (approved):
looks good! thank you

Seeing as this is your first time contributing, if you would like a holographic contributors-only Atuin sticker, then please fill out this [form!](https://notionforms.io/forms/contributors-stickers)

We do also have a [Discord](https://discord.gg/Fq8bJSKPHh) if you'd like to ask any questions, or just fancy hanging out!
